kusto/query_builder.go

```diff
@@ -506,6 +507,185 @@ func NewStmt(query stringConstant, options ...StmtOption) Stmt {
 	return s
 }
 
+// AddDatabase will add a normalized Database string to the Stmt. This is similar to the + operator on two strings,
+// except it only can be done with string constants. This allows dynamically building of a query from a root Stmt.
+func (s Stmt) AddDatabase(query string) Stmt {
+	return s.NormalizeName(query, false)
+}
+
+// AddTable will add a normalized Table string to the Stmt. This is similar to the + operator on two strings,
+// except it only can be done with string constants. This allows dynamically building of a query from a root Stmt.
+func (s Stmt) AddTable(query string) Stmt {
+	return s.NormalizeName(query, false)
+}
+
+// AddColumn will add a normalized Column string to the Stmt. This is similar to the + operator on two strings,
+// except it only can be done with string constants. This allows dynamically building of a query from a root Stmt.
+func (s Stmt) AddColumn(query string) Stmt {
+	return s.NormalizeName(query, false)
+}
+
+// AddFunction will add a normalized Function string to the Stmt. This is similar to the + operator on two strings,
+// except it only can be done with string constants. This allows dynamically building of a query from a root Stmt.
+func (s Stmt) AddFunction(query string) Stmt {
+	return s.NormalizeName(query, false)
+}
+
+// NormalizeName normalizes a string in order to be used safely in the engine.
+func (s Stmt) NormalizeName(query string, forceNormalization bool) Stmt {
+	if query == "" {
+		return s
+	}
+	if !forceNormalization && IsIdentifier(query) {
+		return s
+	}
+	if strings.HasPrefix(query, "[") {
```

Comment:
Drop this code below and always pass through AddQuotedString then surround in []